core/tee/tee_supp_plugin_rpc.c

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Open Mobile Platform LLC
+ */
+
+#include <assert.h>
+#include <kernel/thread.h>
+#include <optee_rpc_cmd.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
+#include <tee/tee_supp_plugin_rpc.h>
+#include <tee/uuid.h>
+#include <trace.h>
+#include <mm/mobj.h>
+
+TEE_Result tee_invoke_supp_plugin_rpc(struct tee_supp_plugin_rpc_op *op)
```

Comment:
Why is this function taking a `struct tee_supp_plugin_rpc_op` only used for this purpose instead of separate parameters?

Reply:
I think we can use separate parameters here. Ok, I will drop `tee_supp_plugin_rpc_op`